open-source-trumps-innersource.md

```diff
@@ -0,0 +1,35 @@
+## Title
+Open Source trumps InnerSource
+
+## Statement of Problem
+* People find the InnerSource project but, after all things are considered, even if the InnerSource component meets their needs, they still go with the open source component.
+
+## Context
+* People look for software through external search engines.
+* Different Business Lines (BLs) are selecting different SW components to do the same function, resulting in inconsistencies in the user experience.
+
+## Forces 
+* There is a perception that the open source components are higher quality and more reusable. The channels to get any needed changes are more obvious with open source than with internal components. Can easily grab the open source whereas internally developed component has less visibility and ease of availability. (This could be separate Pattern). All things equal, they gravitate towards open source.
+* Open source should be voluntary; mandating people to use internally developed software goes against open source principles.
+* Internally developed components should have advantages over competing open source. If you can clearly demonstrate this, it can be persuasive.
```

Comment:
This line seems like part-`Context` (`Internally developed components should have advantages over competing open source`) and part-`Solution` (`If you can clearly demonstrate this, it can be persuasive.`) rather than `Forces`.

Reply:
@rrrutledge how about: Internally developed components can be made more robust and better suited towards corporate needs. Reasons for preferring inner source over open source alternatives might include issues with the open source license, use of proprietary and/or patented components, and functional/competitive advantages. The more that such internal components are used within a company, the better the ROI for the inner source development efforts. On the other hand, open source advances can sometimes leap frog internal development and without periodic assessment, it can be difficult to know when to stop investment in inner source development and switch to the adoption of open source components.

Reply:
I cannot tell if this thread has been worked into the latest version of the pattern already. Will leave the tread open.